src/main/java/teammates/ui/controller/InstructorStudentCommentAddAction.java

```diff
@@ -59,9 +62,9 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         try {
             CommentAttributes createdComment = logic.createComment(comment);
             //TODO: move putDocument to Task Queue
-            logic.putDocument(createdComment);
```

Comment:
Do not remove this line; it's used for enabling search to be performed on student comments.